VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx

```diff
@@ -0,0 +1,143 @@
+import { StackScreenProps } from '@react-navigation/stack'
+import { map } from 'underscore'
+import { useSelector } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+import React, { FC } from 'react'
+
+import { BackgroundVariant, BorderColorVariant, Box, BoxProps, LargePanel, TextView, VAIcon } from 'components'
+import { BranchesOfServiceConstants, ServiceData } from 'store/api/types'
+import { DisabilityRatingState, MilitaryServiceState, PersonalInformationState } from 'store/slices'
+import { HomeStackParamList } from '../HomeStackScreens'
+import { NAMESPACE } from 'constants/namespaces'
+import { RootState } from 'store'
+import { getBirthDate } from '../ProfileScreen/PersonalInformationScreen/PersonalInformationScreen'
+import { useHasMilitaryInformationAccess } from 'utils/authorizationHooks'
+import { useTheme } from 'utils/hooks'
+// import PhotoUpload from 'components/PhotoUpload'
+
+type VeteranStatusScreenProps = StackScreenProps<HomeStackParamList, 'VeteranStatus'>
+
+const VeteranStatusScreen: FC<VeteranStatusScreenProps> = () => {
+  const { profile } = useSelector<RootState, PersonalInformationState>((state) => state.personalInformation)
+  const { serviceHistory, mostRecentBranch } = useSelector<RootState, MilitaryServiceState>((s) => s.militaryService)
+  const { ratingData } = useSelector<RootState, DisabilityRatingState>((state) => state.disabilityRating)
+  const accessToMilitaryInfo = useHasMilitaryInformationAccess()
+  const theme = useTheme()
+  const { t } = useTranslation(NAMESPACE.COMMON)
+
+  const name = (): string => {
+    return profile?.fullName || ''
+  }
```

Comment:
`name` is only used once so probably doesn't need to be a function

Reply:
removed.